index.html

```diff
@@ -33,6 +33,9 @@
       padding-top: 80px;
       padding-bottom: 20px;
     }
+    .github {
+      color: white;
+    }
```

Comment:
Because it would be otherwise blue. Blue looks bad on that red background in the corner.